src/extensions/renderer/base/load-listeners.js

```diff
@@ -698,6 +698,7 @@ BRp.load = function(){
 
         }
 
+        cy.emit('dragpan');
```

Comment:
Should be after delta is applied (i.e. so can read value in handler)

Reply:
It's been moved.